packages/test-utils/lib/test-utils.js

```diff
@@ -15,11 +15,14 @@ async function setupServer({
   createLists = () => {},
   keystoneOptions,
   graphqlOptions = {},
+  dbName,
```

Comment:
Do we still need this `dbName` support? If not, let's revert the relevant changes.